interceptor/src/stats/interceptor.rs

```diff
@@ -155,39 +503,37 @@ impl RTPReader for RTPReadRecorder {
         let mut b = &buf[..bytes_read];
         let packet = rtp::packet::Packet::unmarshal(&mut b)?;
 
-        self.stats.update(
```

Comment:
This used to be based on a collection of `Arc<Atomic*>`s, but this method is brittle because there is not synchronisation between each value. With the new approach a spawned task(one per PC) is in charge of reducing stats and no incomplete updates can be observed.